### PR TITLE
Report view and table to show reports with styles

### PR DESCRIPTION
**Pull request by M-Cordova-M**
# Description

What did you implemented/Why did you implemented this:
We added a report view and a table to show the reports, we also made some changes on employee and stores table styles.

## Screenshots

reloj 5

reloj 6

### CODE REVIEW

**Review by Reiyirak — APPROVED** [review]
Try to remove the development.log, maybe the route is wrong on the gitignore.

**Review by MaxRetana — APPROVED** [review]
LGTM

File: app/views/employees/index.html.erb [hunk 1]
```diff
@@ -35,10 +35,9 @@
       </table>
     </div>
   <% else %>
-    <div class="p-3 mb-2 bg-light text-dark">
+    <div class="p-3 mb-2 bg-dark text-light">
       <span class="text-info font-weight-bold">Welcome! </span>
-      Create your first post
+      Register an employee
```
Comment (juperameza) [inline app/views/employees/index.html.erb:40]: Standardize your language you have some things in English and some  in Spanish

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by dagp-ucol — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**